addons/fire/initSettings.sqf

```diff
@@ -0,0 +1,19 @@
+[
+    QGVAR(enabled), "CHECKBOX",
```

Comment:
One is "enabled", the other (flare) is "enable". Not sure what we use most of the time (former I think), but both should be the same.

Settings are really the only thing I want double-checked before merge, so we can avoid changing them later if at all possible.